yarn-project/circuit-types/src/stats/stats.ts

```diff
@@ -1,3 +1,15 @@
+/** Stats associated with an ACIR proof generation.*/
+export type ProofConstructed = {
+  /** Name of the event for metrics purposes */
+  eventName: 'proof_construction_time';
+  /** Name of the program being proven */
+  acir_test: string;
+  /** Number of threads used for proving */
+  threads: number;
+  /** Time spent proving */
+  value: number;
+};
```

Comment:
Heads up that @alexghr and @PhilWindle were working on similar benchmarks I believe

Reply:
Yep, see [this comment](https://github.com/AztecProtocol/aztec-packages/pull/6409#issuecomment-2115572947) for the new metrics tracked per private function. The "duration" stat in PR #6409 includes time spent spawning the bb process and its IO.

Reply:
I think that these two benchmarks are sufficiently different to have both. @alexghr's focuses on proving benchmarks for Noir contracts, whereas this PR is set up to track regular Noir programs to benchmark against circom.